lib/puma/dsl.rb

```diff
@@ -308,7 +308,7 @@ def debug
     # @example
     #   rackup '/u/apps/lolcat/config.ru'
     def rackup(path)
-      @options[:rackup] = path.to_s
+      @options[:rackup] = path.to_s unless @options[:rackup]
```

Comment:
```suggestion
      @options[:rackup] ||= path.to_s
```
Perhaps even simpler?

Reply:
@cristian-rivera I've tried this approach on [26393ae](https://github.com/puma/puma/pull/2236/commits/26393ae933dcad90510348db9033b14a67075e79) but the tests failed on two builds, so I reverted the change.

Reply:
These suggested changes -> commit things are so nice.